src/sagas/session.js

```diff
@@ -83,7 +83,11 @@ export function* listBuckets(getState, action) {
     // If the Kinto API version allows it, retrieves all permissions
     if ("permissions_endpoint" in serverInfo.capabilities) {
       const {data: permissions} = yield call([client, client.listPermissions]);
-      buckets = handlePermissions(buckets, permissions);
+      buckets = expandBucketsCollections(buckets, permissions);
+      yield put(sessionActions.permissionsListSuccess(permissions));
+    }
+    else {
+      yield put(notificationActions.notifyInfo("Permissions endpoint is not enabled on server."));
```

Comment:
I can't figure out good wording. And don't know a clean way to put some html (link) in the notification message (from saga, outside react views)

This could land like this until someone gives a hand ;)